### PR TITLE
Feature/localization

### PR DESCRIPTION
**Pull request by CibelePaulinoAndrade**
Finishing the localization of the app

### CODE REVIEW

File: FinalChallenge/Utils/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -20,15 +22,35 @@
...
+
 // Texts related to alerts
 "closeAlertText" = "Are you sure you want to close? Your progress will be lost.";
 "closeAlertButtonText" = "Yes";
+"closeAndSaveAlertText" = "You have changes not saved. Do you like to save them?";
 "saveAlertText" = "Task saved. Successfully redone schedule!";
 "saveAlertButtonText" = "OK!";
+"saveRoomAlertText" = "Enter title for room...";
+"placeHolderSaveRoomAlertText" = "eg: Bathroom";
+"saveDailyTime" = "How much time (in minutes) per day would you like to spend tidy up your place?";
```
Comment (jessicalewinter) [inline FinalChallenge/Utils/Localization/en.lproj/Localizable.strings]:
```suggestion
"saveDailyTime" = "How much time (in minutes) per day would you like to spend tidying up your place?";
```

File: FinalChallenge/Utils/Localization/es.lproj/Localizable.strings [hunk 2]
```diff
@@ -20,15 +22,35 @@
...
+"dailyTimeText" = "Tiempo Diario";
+"loading" = "Cargando";
+
 // Texts related to alerts
 "closeAlertText" = "¿Estás seguro de que quieres cerrar? Tu progreso se perderá.";
 "closeAlertButtonText" = "Si";
+"closeAndSaveAlertText" = "Tienes cambios sin guardar. ¿Te gustaría salvarlos?";
 "saveAlertText" = "Tarea guardada. Horario exitosamente renovado!";
 "saveAlertButtonText" = "Cierto!";
+"saveRoomAlertText" = "Ingrese el título de la habitación..";
```
Comment (jessicalewinter) [inline FinalChallenge/Utils/Localization/es.lproj/Localizable.strings]:
```suggestion
"saveRoomAlertText" = "Ingrese el título de la habitación.";
```

File: FinalChallenge/Utils/Localization/pt-BR.lproj/Localizable.strings [hunk 3]
```diff
@@ -20,15 +22,35 @@
...
 
+// texts related to AllTasks View
+"dailyFrequency" = "Diariamente";
+"weeklyFrequency" = "vezes na semana";
+"monthlyFrequency" = "Mensalmente";
+"hour" = "hrs";
+"minutes" = "min";
+
+// texts related to Completed Task View
+"completedMessage" = "Boa! tarefa concluída a tempo.";
```
Comment (jessicalewinter) [inline FinalChallenge/Utils/Localization/pt-BR.lproj/Localizable.strings]:
```suggestion
"completedMessage" = "Boa! Tarefa concluída a tempo.";
```

File: FinalChallenge/Utils/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Utils/Localization/en.lproj/Localizable.strings]: done

File: FinalChallenge/Utils/Localization/es.lproj/Localizable.strings (hunk 2 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Utils/Localization/es.lproj/Localizable.strings]: done

File: FinalChallenge/Utils/Localization/pt-BR.lproj/Localizable.strings (hunk 3 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Utils/Localization/pt-BR.lproj/Localizable.strings]: Done

**Pull request merged**